src/Blt/Plugin/Commands/CodeceptionCommands.php

```diff
@@ -105,6 +116,10 @@ protected function runCodeceptionTestSuite($suite, $test_directory) {
       $test->option('env', 'ci', '=');
     }
 
+    if ($this->failFast) {
+      $test->option('fail-fast', $this->failFast, '=');
```

Comment:
```suggestion
      $test->option('fail-fast');
```

Reply:
I was trying to keep the entire configuration intact.
See: https://codeception.com/docs/reference/Commands
``--fail-fast (-f)       Stop after nth failure (defaults to 1)``

For example, `--fail-fast=2` would need 2 failures before stopping.

Reply:
Oh well that's neat. I didn't know it accepted numbers. Lets make sure we cast it then. because you could do `--fail-fast=foobar`

Reply:
Do you want me to just cast it like `(int) $this->failFast` or type it as an `int` with validation?

Reply:
hmm. probably just the former. it'll be easy enough. maybe do it when setting `$this->failFast = (int) $options['fail-fast']` ?

Reply:
Done.